person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  # Constructor
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  private
+
+  def can_use_service?
+    @parent_permission || ofage?
+  end
+
+  def ofage?
```

Comment:
- Kindly to a naming convention. Fix the name of this method to follow the snake case convention as it's used in Ruby codes. It will be some like this `of_age?` Remember to change it whenever you are calling the method